app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/MessageDetailViewModel.java

```diff
@@ -35,6 +36,10 @@ private boolean isSelectedMessageOfType(@Message.MessageType String messageType)
         return mSelectedMessageModel.getSelected().getType().equals(messageType);
     }
 
+    public void drawMessageOnMap(MessagesDetailBaseGeoFragment.GeoMessageInterface drawMessageOnMapInterface) {
```

Comment:
View-model should be unaware of the view.
This interface functionality should be defining at the View-Model tier !fix